src/app.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const fs = require('fs');
+const datas = process.argv.slice(2);
+
+if (datas[0] === 'cp') {
+  copyFile(datas[1], datas[2]);
+}
+
+function copyFile(file, copy) {
```

Comment:
Add a check to ensure that the file and copy parameters are not undefined or empty strings.